packages/ruby.rb

```diff
@@ -3,12 +3,11 @@
 class Ruby < Package
   description 'Ruby is a dynamic, open source programming language with a focus on simplicity and productivity.'
   homepage 'https://www.ruby-lang.org/en/'
-  @_ver = '3.1.2'
-  version "#{@_ver}-2"
+  version = '3.1.2-2'
   license 'Ruby-BSD and BSD-2'
   compatibility 'all'
   source_url 'https://github.com/ruby/ruby.git'
-  git_hashtag "v#{@_ver.tr('.', '_')}"
```

Comment:
Why?

Reply:
So I could extract the version via sed without having a horrifying mess of pattern matching.

Reply:
Why not just take the version from @_ver and replace `.` with `_` ? (This makes updating the version in the package files easier.)

You don't need to use sed to extract the version from git_hashtag...

Reply:
(In fairness I have a high complicated sed section in my build script for handling `@_ver` so I can't blame you... :p

Reply:
So we've been using the `@_ver` to be the version for the source package and `version` for the package version in a bunch of packages to avoid that craziness.

Reply:
Ah. I kinda need to have the version in version, otherwise it sorta breaks the for loop amongst other things, so not sure what to do here.